WordPress/Classes/ViewRelated/NewGutenberg/NewGutenbergViewController.swift

```diff
@@ -789,7 +789,6 @@ extension NewGutenbergViewController {
             return // TODO: when can it happen?
         }
         service.fetchSettings({ [weak self] result in
-            guard let `self` = self else { return }
```

Comment:
This would require the commented line in the TODO below to use `self?` instead of `self?`, but I guess the compiler will remind us of that if/when the time comes.